tests/test_extract.py

```diff
@@ -21,192 +21,199 @@ def test_group_sequence(self):
 
     def test_hpo(self):
 
-        # Test extracting an abbreviated phenotype
-        truth = json.dumps([{"hpid": ["HP:0001370"], "index": [0, 2], "matched": "RA"}])
-        self.assertEqual(hpo("RA"), truth)
+        hpo = Extractor(correct_spelling=False).hpo
```

Comment:
suggesting changing this to `extractor = Extractor(correct_spelling=False)`. Then use `extractor.hpo(term)` later